### ISSUE TITLE
Duplicated rules in Prometheus - "Prometheus is missing rule evaluations due to slow rule group evaluation"

### ISSUE THREAD

**Issue by MKlimuszka** [rancher/rancher#40255]
Rancher Cluster:
Rancher version: 2.6.6
Number of nodes: 3
Node OS version: Ubuntu 20.04

Downstream Cluster:
Number of Downstream clusters: 2
Node OS: Ubuntu 20.04
RKE/RKE2/K3S version:
Kubernetes version: 1.22.10
CNI: Canal

Longhorn:
Longhorn version: N/A
CPU per node:  N/A
Memory per node: N/A
Disk type: HDD/SSD/NVMe
Network bandwidth between the nodes: N/A

Other:
Underlying Infrastructure: AWS/GCE, EKS/GKE, VMWare/KVM, Baremetal

Issue description:
Rancher Monitoring alert for "Prometheus is missing rule evaluations due to slow rule group evaluation".

According to [this GitHub issue](https://github.com/prometheus-community/helm-charts/issues/1799), there was a change to the Helm charts for Prometheus which moved some rules into their own separately grouped rule files but left the original rules in place, thus creating duplicate rules.

Business impact:
False alerts in Prometheus due to duplicated rules

Other:
SURE-5069

**Comment by MKlimuszka** [comment]
This needs to be reproduced in 2023-Q1-2.6x, 2023-Q1-2.7x, and if it cannot be reproduced in those, 2.6.6

**Comment by PennyScissors** [comment]
The duplicated rules were removed upstream in [this PR](https://github.com/prometheus-community/helm-charts/pull/2076) as part of the kube-prometheus-stack version 35.3.2. Rancher's Monitoring chart is currently at upstream version 40.1.2, therefore, this fix is present in our latest chart. I verified this by looking at the Prometheus rules in both our chart's files [here](https://github.com/rancher/charts/tree/dev-v2.7/charts/rancher-monitoring/102.0.0%2Bup40.1.2/templates/prometheus/rules-1.14) (file `templates/prometheus/rules-1.14/kube-apiserver.rules.yaml` no longer exists, this is correct) and the Prometheus Rules UI (`kube-apiserver.rules` no longer exists, this is correct) in a Rancher v2.7-head 9376ad2 setup with Monitoring 102.0.0+up40.1.2.
